pkgs/development/interpreters/babashka/wrapped.nix

```diff
@@ -44,5 +43,13 @@ stdenvNoCC.mkDerivation (finalAttrs: {
         --replace '"${unwrapped-bin}"' '"${rlwrap}/bin/rlwrap" "${unwrapped-bin}"'
     '';
 
+  installCheckPhase = ''
+    ${babashka-unwrapped.installCheckPhase}
+    # Needed for Darwin compat, see https://github.com/borkdude/deps.clj/issues/114
+    export CLJ_CONFIG="$TMP/.clojure"
+    $out/bin/bb clojure --version | grep -wF '${clojureToolsBabashka.version}'
+  '';
```

Comment:
This check looks like it broke the @ofborg build, can you remove it?

Reply:
Interesting, I would add this to the `buildGraalvmNativeImage` instead of just babashka (and for all environments, not just `x86_64-darwin`).

Reply:
According to https://www.graalvm.org/latest/reference-manual/native-image/overview/BuildOptions/ , the `-march` option impacts the performance of the native executable. I agree that makes sense to add it to  `buildGraalvmNativeImage`, but I wouldn't add to all environments.

It could be interesting look at Babashka performance with different `march` values.

Reply:
That makes sense. How about adding it in a way so that it's still overridable if desired?

Reply:
Nevermind, I'll just add it to the default `nativeImageBuildArgs` which sufficiently supports that case I'd say.

Reply:
Made a separate commit for that change - or would it be preferable to break it out into a separate PR even?

Reply:
A separate commit is fine.